`no-multi-spaces` conflicts with `smart-tabs`

If I have `"no-mixed-spaces-and-tabs":[2, "smart-tabs"]`, then `no-multi-spaces` should not throw an error/warning for lines that use smart-tabs.

test case, try with the following code:
```
/*eslint no-mixed-spaces-and-tabs:[2, "smart-tabs"], no-multi-spaces:2 */
var i=10,
    j=20;
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/9628407-no-multi-spaces-conflicts-with-smart-tabs?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

0.17.0

```
   441:7   warning  Multiple spaces found before 'i'                  no-multi-spaces
   442:7   warning  Multiple spaces found before 'gotcookies'         no-multi-spaces
   443:7   warning  Multiple spaces found before 'cookies'            no-multi-spaces
```

code:
```
			var cookies_a,
			    i, l, kv,
			    gotcookies=false,
			    cookies={};
```
(3 tabs, and then 4 spaces to align variable names)

--- a/lib/rules/no-multi-spaces.js
+++ b/lib/rules/no-multi-spaces.js
@@ -70,7 +70,7 @@ module.exports = function(context) {
 
             var source = context.getSource(),
                 allComments = context.getAllComments(),
-                pattern = /[^\n\r\u2028\u2029 ] {2,}/g,  // note: repeating space
+                pattern = /[^\n\r\u2028\u2029\t ].? {2,}/g,  // note: repeating space
                 token,
                 previousToken,
                 parent;

--- a/tests/lib/rules/no-multi-spaces.js
+++ b/tests/lib/rules/no-multi-spaces.js
@@ -26,6 +26,7 @@ ruleTester.run("no-multi-spaces", rule, {
         "var arr = [1, 2];",
         "var arr = [ (1), (2) ];",
         "var obj = {'a': 1, 'b': (2)};",
+        "\t\tvar x = 5,\n\t\t    y = 2;",
         "a, b",
         "a >>> b",
         "a ^ b",
@@ -361,6 +362,22 @@ ruleTester.run("no-multi-spaces", rule, {
             errors: [{
                 message: "Multiple spaces found before '+'.",
                 type: "Punctuator"
+            }]
+        },
+        {
+            code: "\t\tvar x = 5,\n\t\t    y =  2;",
+            output: "\t\tvar x = 5,\n\t\t    y = 2;",
+            errors: [{
+                message: "Multiple spaces found before '2'.",
+                type: "Numeric"
+            }]
+        },
+        {
+            code: "var x =\t  5;",
+            output: "var x = 5;",
+            errors: [{
+                message: "Multiple spaces found before '5'.",
+                type: "Numeric"
             }]
         }
     ]

